dockerfiles/init/manifests/codenvy.env

```diff
@@ -86,17 +86,121 @@
 #CODENVY_SERVER_XMX=2048
 
 
-
 ########################################################################################
 #####                                                                              #####
-#####                                   DOCKER                                     #####
+#####                                 NETWORKING                                   #####
 #####                                                                              #####
-# IP Address
-#     The IP address of the Docker daemon that is running on your host. We do a 
-#     self-discovery to set this. You can combine this with values of DOCKER_HOST to 
-#     change communications from socket to TCP.
+#
+# Affects how the Codenvy server connects to the Docker Daemon, how the Che server and
+# workspaces establish connections to one another, and also how remote browser clients
+# should discover and connect to workspaces.
+#
+# Codenvy goes through a progression algorithm to establish the protocol, IP address and
+# port to establish communications with it is booting or starting a workspace.
+#
+# Browser --> Codenvy Server
+#    1. Default is 'http://localhost:${SERVER_PORT}/wsmaster/api'.
+#    2. Else use the value of CODENVY_API
```

Comment:
No, I meant CODENVY_API.  In Che, we actually have a CHE_API= variable that is the URL of the entrypoint.  I suspect that with Codenvy no such variable exists, as it's internally coded and not something we should expose.

in the case of CODENVY, we have the reverse proxy, so it probably is CODENVY_HOST, but I need someone to tell me what actually happens and then can update content.